content/tutorials/token-authentication.textile

```diff
@@ -448,6 +620,9 @@ blang[nodejs].
 blang[ruby].
   Ok, we're ready to see all of this in action. Restart the local server @bundle exec ruby server.rb@ and then refresh the page at "http://localhost:4567/":http://localhost:4567/. You should receive an alert to confirm that you're connected as an anonymous with limited capabilities as follows:
 
+blang[ruby].
```

Comment:
I am concerned that you have not tried this tutorial documentation out as this would not have worked.  Have you been running the server using Docker as instructed?